allensdk/brain_observatory/nwb/__init__.py

```diff
@@ -774,12 +795,19 @@ def add_cell_specimen_table(nwbfile, cell_specimen_table):
         description="Segmented rois",
         imaging_plane=imaging_plane)
 
-    for c in [c for c in cell_roi_table.columns if c not in ['id', 'mask_matrix']]:
-        plane_segmentation.add_column(c, c)
+    for col_name in cell_roi_table.columns:
+        # the columns 'image_mask', 'pixel_mask', and 'voxel_mask' are already defined
+        # in the nwb.ophys::PlaneSegmentation Object
+        if col_name not in ['id', 'mask_matrix', 'image_mask', 'pixel_mask', 'voxel_mask']:
+            # This builds the columns with name of column and description of column
+            # both equal to the column name in the cell_roi_table
+            plane_segmentation.add_column(col_name, col_name)
```

Comment:
The `add_column()` method is inherited from hdmf. I think the pynwb devs would probably there to be an added description for what each column is (see the `description param` in the method:

https://hdmf.readthedocs.io/en/latest/hdmf.common.table.html#hdmf.common.table.DynamicTable

Reply:
The column name is the description in this case, I'll add some extra descriptions for each column.

Reply:
I added the column names in the __init__.py file, check the names out and let me know if you would change anything.